clang/lib/Sema/SemaDecl.cpp

```diff
@@ -2650,6 +2658,7 @@ void Sema::MergeTypedefNameDecl(Scope *S, TypedefNameDecl *New,
         Scope *EnumScope = getNonFieldDeclScope(S);
         for (auto *D : NewTag->decls()) {
           auto *ED = cast<EnumConstantDecl>(D);
+         llvm::outs()<<"@@@@"<<ED->getName()<<"%%%%";
```

Comment:
run git-clang-format for proper indentations

Reply:
Do you know if this is the right place to make your changes? this function seems to affect Typedef. Ignore if this change is required and works correctly.